lnrpc/rpc.proto

```diff
@@ -3461,6 +3472,18 @@ message BakeMacaroonResponse {
     string macaroon = 1;
 }
 
+message ListSweepsRequest {
+}
+
+message ListSweepsResponse {
+    /*
+    The hex encoded transaction ids of the sweeps that our node has broadcast.
+    Note that these transactions may not have confirmed yet, we record sweeps
+    on broadcast, not confirmation.
+    */
+    repeated string transaction_ids = 1;
```

Comment:
I'm not familiar with the context of this feature (maybe this is requested as implemented here), but perhaps we could instead return a list of transactions ordered by height. I believe that is more useful than just a random ordered tx hash list. We can do this as that is the intersection of `GetTransactions` and `ListSweeps`. WDYT?

Reply:
I'm guessing we don't have this information in the sweeper. The transaction information itself will be in the `GetTransactions` call, this output allows RPC users to identify _which_ of these transactions were sweeps.

Reply:
> return a list of transactions ordered by height

My first run at this had a `--verbose` option which would lookup all the sweep txns if true, and just list the txids if it was false. I removed it because it felt unnecessary to have a special case `GetTransactions` call, but that could be an option here @bhandras?

Another thing we could do is leave it just as transaction ID for now and start storing broadcast height hints for sweeps so that `GetTransactions` can be more efficiently called to lookup these sweeps.

Reply:
I like the `--verbose` idea.